packages/conventional-changelog-writer/index.js

```diff
@@ -30,7 +30,11 @@ function conventionalChangelogWriter (context, options) {
     noteGroupsSort: 'title',
     notesSort: 'text',
     generateOn: function (commit) {
-      return semverValid(commit.version)
+      var semverOptions = {}
+      if (options.looseSemver) {
+        semverOptions = { loose: true }
```

Comment:
Does this check that `options.looseSemver` is not `null`?

Could you do this in one line with `semverOptions = { loose: options.looseSemver ? true : false }`

Reply:
It doesn't - it'll just fall out and `semverOptions` will never be set which is null-safe in the semver package where the options are passed. Happy to clean it up though